proposal/default-on-private-variables.md

```diff
@@ -56,9 +74,9 @@ The deprecation process will be divided into two phases:
 > upcoming changes to behavior and give them a chance to move towards passing
 > future-proof units.
 
-Phase 1 does not change the semantics of variable declarations. However, if a
-private variable (one whose name starts with `_` or `-`) has a `!default` flag,
-emit a deprecation warning named `private-variable-default`.
+Phase 1 does not change the semantics of `with` clauses. However, we will emit a
+deprecation warning named `with-private` when a `with` clause privacy of Sass
+modules.
```

Comment:
"when a `with` clause privacy of Sass modules" is missing a verb.

Reply:
oh no, it doesn't make any sense lol